neptune/codelistsdataset_test.go

```diff
@@ -108,20 +108,20 @@ func TestBuildResponse(t *testing.T) {
 					codeListID := "testCodeListID"
 					response := buildResponse(did2Dim, codeValue, codeListID)
 					Convey("Then the response should be well formed", func() {
-						So(response.Count, ShouldEqual, 2)
+						// So(response.Count, ShouldEqual, 2)
```

Comment:
remove commented out code or put TODO as to when it should come back in

Reply:
I will remove, count is not part of the data structure anymore